packages/react-rtc/src/hooks/usePeerConnection.ts

```diff
@@ -28,33 +28,33 @@ export const usePeerConnection = (
     dispatchEvent('leave');
   };
 
-  const onIceCandidate = (
-    event: RTCPeerConnectionIceEvent,
-    peerUuid: string
-  ) => {
-    if (event.candidate)
-      sendSignalingMessage(peerUuid, { ice: event.candidate });
+  const onIceCandidate = (event: RTCPeerConnectionIceEvent, peerId: string) => {
+    if (event.candidate) sendSignalingMessage(peerId, { ice: event.candidate });
   };
 
-  const addNewPeer = (peerUuid: string, initCall = false) => {
+  const addNewPeer = (peerId: string, initCall = false) => {
     const peerConnection = new RTCPeerConnection({ iceServers });
     const dataChannel = peerConnection.createDataChannel(crypto.randomUUID());
 
     peerConnection.addEventListener('icecandidate', (event) =>
-      onIceCandidate(event, peerUuid)
+      onIceCandidate(event, peerId)
     );
     peerConnection.addEventListener('iceconnectionstatechange', () =>
-      checkPeerDisconnect(peerUuid)
+      checkPeerDisconnect(peerId)
     );
     peerConnection.addEventListener('datachannel', (event) =>
-      Object.defineProperty(peerConnections.get(peerUuid), 'dataChannel', {
+      Object.defineProperty(peerConnections.get(peerId), 'dataChannel', {
         value: event.channel,
       })
     );
     peerConnection.addEventListener('connectionstatechange', () => {
-      const peer = peerConnections.get(peerUuid);
-      if (peer && peer.pc.connectionState === 'connected' && !initCall)
-        dispatchEvent('peerConnected', peer);
+      const peer = peerConnections.get(peerId);
+      if (
+        peer &&
+        peer.peerConnection.connectionState === 'connected' &&
+        !initCall
+      )
+        dispatchEvent('peerConnected', peer.id);
```

Comment:
Would recommend:
- Add descriptive variable
- Avoid `&&`
- Use brackets for consistency

```suggestion
      const peer = peerConnections.get(peerId);
      const isConnected = peer?.peerConnection.connectionState === 'connected'
      
      if (isConnected && !initCall) {
        dispatchEvent('peerConnected', peer.id);
      }
```

Reply:
Indeed better! I am also thinking about making function `getConnectionState` or `isConnected` inside `usePeers` hook https://github.com/torolocos/react-rtc/blob/0b524facc0bf75e5d37bbbb2b910a982ad208893/packages/react-rtc/src/hooks/usePeers.ts#L4.